index.html

```diff
@@ -12,107 +12,67 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
-  <header>
-    <nav>
-      <a href="/">My Logo</a>
-      <button onclick="return;" type="button">
-        <span>
-          <img
-            src="assets/images/icons/Union.png" 
-            alt="nav button">
-        </span>
-      </button>
-    </nav>
+  <header class="header">
+      <nav class="navbar">
+         <a class="header-logo" href="#">My Logo</a>
+         <div class="menu-icon">
+          <a class="menubar" href="#">
+            <img src="assets/images/icons/menu-button.png" 
+              alt="menu" />
+          </a>
+         </div>
+      </nav>
   </header>
```

Comment:
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/60083437/15f4e2bd-2235-47c6-a873-fdca5c705c86)

- [ ] `Logo` and `Menu` should be aligned with each other. But here `Logo` seems a bit below to the `Menu`. Kindly fix this issue to match the given template.